tmt.spec

```diff
@@ -79,7 +79,7 @@ Summary: Extra dependencies for the Test Management Tool
 Requires: tmt >= %{version}
 Requires: tmt-container >= %{version}
 Requires: tmt-testcloud >= %{version}
-Requires: vagrant python3-nitrate make
+Requires: python3-nitrate make
```

Comment:
```suggestion
Requires: python3-nitrate make
%if %{with vagrant}
Recommends: vagrant
%endif
```

Reply:
Moved `vagrant` to `Recommends`. Let's do any further spec changes in the vagrant dynamic plugin pull request.